src/Files.Shared/Helpers/FileExtensionHelpers.cs

```diff
@@ -31,10 +31,20 @@ public static bool HasExtension(string? filePathToCheck, params string[] extensi
 		/// <param name="fileExtensionToCheck">The file extension to check.</param>
 		/// <returns><c>true</c> if the fileExtensionToCheck is an image; otherwise, <c>false</c>.</returns>
 		public static bool IsImageFile(string? fileExtensionToCheck)
+		{
+			return HasExtension(fileExtensionToCheck, ".png", ".bmp", ".jpg", ".jpeg", ".jfif", ".gif", ".tiff", ".tif", ".webp");
+		}
+
+		/// <summary>
+		/// Checks if the file can be set as wallpaper.
+		/// </summary>
+		/// <param name="fileExtensionToCheck">The file extension to check.</param>
+		/// <returns><c>true</c> if the fileExtensionToCheck is an image; otherwise, <c>false</c>.</returns>
+		public static bool IsManipulateableImageFile(string? fileExtensionToCheck)
```

Comment:
```suggestion
		public static bool IsCompatibleToSetAsWindowsWallpaper(string? fileExtensionToCheck)
```

Reply:
Sure.
FYI, I think when I refactor WallpaperHelper to be a servive called `WindowsWallpaperService`, I would add a method called `IsValidFileExtension(string extension)` to check if being a valid image file or not.

Reply:
Done